airflow/contrib/executors/kubernetes_executor.py

```diff
@@ -252,6 +252,15 @@ def __init__(self):
         else:
             self.kube_tolerations = None
 
+        kube_client_request_args = conf.get(self.kubernetes_section, 'kube_client_request_args')
+        if kube_client_request_args:
+            self.kube_client_request_args = json.loads(kube_client_request_args)
+            if self.kube_client_request_args['_request_timeout'] and \
```

Comment:
value for request timeout can be supplied as array(connect and read timeouts) , since json.loads converts arrays to list, we would need to change it to tuple -- to be used here- https://github.com/kubernetes-client/python/blob/7fb3a25176395da26007aec7b4c843aa03514c21/kubernetes/client/rest.py#L143